cloud/errors.go

```diff
@@ -50,3 +55,33 @@ func (e cpiError) Message() string {
 func (e cpiError) OkToRetry() bool {
 	return e.cmdError.OkToRetry
 }
+
+func mapsToNotImplementedError(method string, cmdError CmdError) bool {
+	matched, _ := regexp.MatchString("^Method .+ not supported in photon CPI", cmdError.Message)
```

Comment:
We would suggest to omit `photon`, e.g. `/^Method .+ not supported.*/`. We also need to keep this in sync with the director, where this has already been [implemented](https://github.com/cloudfoundry/bosh/blob/f40d35342ce8cd989ffa8eb73ef5045c2099ca36/src/bosh-director/lib/cloud/external_cpi.rb#L175).

We would really like to ignore only method missing style errors from cpis to avoid swallowing real errors. What is going to happen in 6 months?